crypto/x509/x509_vfy.c

```diff
@@ -2317,6 +2286,10 @@ int X509_STORE_CTX_init(X509_STORE_CTX *ctx, X509_STORE *store, X509 *x509,
 {
     int ret = 1;
 
+    if (ctx == NULL) {
+        ERR_raise(ERR_LIB_X509, ERR_R_PASSED_NULL_PARAMETER);
+        return 0;
+    }
```

Comment:
This appears unrelated to the PR.

Reply:
Right, it's a "drive-by improvement".

Reply:
I've meanwhile moved this unrelated change to a different PR.